library/pathname/glob_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative '../../spec_helper'
+require 'pathname'
+
+describe 'Pathname#glob' do
+  before :all  do
+    @dir = tmp ''
```

Comment:
This should have a name, e.g. `pathname_glob`, otherwise `tmp ''` will return the spec dir, and that one should not be removed while specs are running.

Reply:
```suggestion
    @dir = tmp 'pathname_glob'
```